yetanotherco/aligned_layer
# feat: pin batcher gnark version

pins batcher gnark version used for verification.

# To test

Start anvil, batcher, aggregator and operator
Send groth16 & plonk proofs to batcher and it should verify them correctly
